### PR TITLE
Bug fixes and enhancements.

### PR DESCRIPTION
**Pull request by mwfong-csl**
See commit history for more info.

### CODE REVIEW

File: ifparser/ifconfig.py [hunk 1]
```diff
@@ -3,15 +3,24 @@
...
-    _attrs = frozenset([
-        'interface', 'itype', 'mtu', 'ip', 'bcast', 'mask', 'hwaddr',
-        'txbytes', 'rxbytes', 'rxpkts', 'txpkts', 'rxdroppedpkts',
-        'rxoverruns', 'rxframe', 'rxerrors'
-    ])
-    _flags = frozenset([
+    _attrList = [
+        'interface', 'itype', 'mtu', 'ip', 'bcast', 'mask', 'hwaddr'
+    ]
+    _cntFieldList = [
```
Comment (ssudake21) [inline ifparser/ifconfig.py]: Use tuple instead of list

File: ifparser/ifconfig.py [hunk 2]
```diff
@@ -3,15 +3,24 @@
 
 
 class Interface(object):
-    _attrs = frozenset([
-        'interface', 'itype', 'mtu', 'ip', 'bcast', 'mask', 'hwaddr',
-        'txbytes', 'rxbytes', 'rxpkts', 'txpkts', 'rxdroppedpkts',
-        'rxoverruns', 'rxframe', 'rxerrors'
-    ])
-    _flags = frozenset([
+    _attrList = [
```
Comment (ssudake21) [inline ifparser/ifconfig.py]: Please use python standards for variable naming.
`_attrList` to `_attr_list`

**Comment by ssudake21** [comment]
@mwfong-csl Minor comments. Please check these.

**Comment by mwfong-csl** [comment]
@ssudake21 'Saw your comments but am currently tied up at work.

**Comment by ssudake21** [comment]
Sure no problem. Take your time.

**Pull request merged**